validator/xsd/schedule-without-person.xml.xsd

```diff
@@ -15,6 +15,7 @@
               <xs:element type="xs:integer" name="days" minOccurs="0"/>
               <xs:element type="duration"   name="timeslot_duration" minOccurs="0"/>
               <xs:element type="httpURI"    name="base_url" minOccurs="0"/>
+	      <xs:element type="xs:string"  name="time_zone_name" minOccurs="0" maxOccurs="1"/>
```

Comment:
Off-by one indentation if this is relevant for the XSD.

Reply:
thanks, fixed